spec/index.bs

```diff
@@ -5296,14 +5299,39 @@ dictionary GPUExternalTextureDescriptor
 
                     Otherwise:
 
-                    1. If |source| <l spec=html>[=is not origin-clean=]</l>,
-                        throw a {{SecurityError}} and return.
+                    1. If |source| is a {{GPUTextureView}}:
+
+                        1. If any of the following conditions are unsatisfied
+                            [$generate a validation error$] and return an [$invalidated$] {{GPUExternalTexture}}.
+
+                            <div class=validusage>
```

Comment:
What if the view is an error view?

Reply:
Thanks for catching! I've added `|source| is [$valid to use with$] |this|.`